test/rest_api/test_compaction_task.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+# Use the util.py library from ../cql-pytest:
+sys.path.insert(1, sys.path[0] + '/../cql-pytest')
+from util import new_test_table, new_test_keyspace
+from rest_util import set_tmp_task_ttl
+
+def wait_for_task(rest_api, task_id):
```

Comment:
since this helper isn't specific to compaction let's find some common place for it so it could be reused in many task_manager related tests.

Reply:
I opened a PR with helpers to all task manager api calls kept together: https://github.com/scylladb/scylladb/pull/12844